.github/requirements/build-requirements.txt

```diff
@@ -81,11 +69,7 @@ semantic-version==2.10.0 \
 setuptools-rust==1.7.0 \
     --hash=sha256:071099885949132a2180d16abf907b60837e74b4085047ba7e9c0f5b365310c1 \
     --hash=sha256:c7100999948235a38ae7e555fe199aa66c253dc384b125f5d85473bf81eae3a3
-     # via -r build-requirements.in
-tomli==2.0.1 \
-    --hash=sha256:939de3e7a6161af0c887ef91b7d41a53e7c5a1ca976325f429cb46ea9bc30ecc \
-    --hash=sha256:de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f
-    # via setuptools-rust
+    # via -r build-requirements.in
```

Comment:
This change here is what caused the test failures btw.

Reply:
Ahh, I ran the `pip-compile` under Python 3.11, so setuptools et al probably decided they didn't need an aftermarket TOML parser.